tests/test_posttest.py

```diff
@@ -29,11 +71,18 @@ def test_collect_techsupport(request, duthosts, enum_dut_hostname):
     # Because Jenkins is configured to save artifacts from tests/logs,
     # and this util is mainly designed for running on Jenkins,
     # save path is fixed to logs for now.
-    TECHSUPPORT_SAVE_PATH = 'logs/'
     out = duthost.command("show techsupport --since {}".format(since), module_ignore_errors=True)
     if out['rc'] == 0:
         tar_file = out['stdout_lines'][-1]
-        duthost.fetch(src=tar_file, dest=TECHSUPPORT_SAVE_PATH, flat=True)
+        tar_file_name = tar_file.split('/')[-1]
+        dump_path = LOG_SAVE_PATH + 'dump/'
+        duthost.fetch(src=tar_file, dest=dump_path, flat=True)
+
+        if not testbed_idx:
+            testbed_idx = tar_file_name
+        tar_file_path = os.path.join(dump_path, tar_file_name)
+        logger.info("tar_file_path: {}, dump_path: {}".format(tar_file_path, dump_path))
+        extract_dump_and_syslog(tar_file_path, dump_path, testbed_idx)
```

Comment:
After the extract, does the original dump file still exist?

Reply:
No, it will be deleted

Reply:
Can we keep it by copy and extract the copied one? Sometime it's convenient for the user to download the dumpfile

Reply:
fixed